vector/src/main/res/layout/item_timeline_event_merged_room_creation_stub.xml

```diff
@@ -65,28 +64,29 @@
         <TextView
             android:id="@+id/roomNameTileText"
             style="@style/Widget.Vector.TextView.Title"
-            android:layout_width="match_parent"
+            android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_marginTop="20dp"
+            android:layout_marginStart="12dp"
             android:layout_marginEnd="16dp"
             android:textColor="?vctr_content_primary"
             android:textStyle="bold"
             app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@id/creationTileRoomAvatarImageView"
+            app:layout_constraintStart_toEndOf="@id/creationTileRoomAvatarImageView"
```

Comment:
It does not work for long room names, the room name overlaps some views, I have to check how to fix it (changing some constraints or adding a max lines property)

Reply:
Fixed by moving the room name below the avatar